README.md

```diff
@@ -32,17 +40,14 @@ Collection of (search) facet, filter, and sort components
 
 #### Usage
 
-Include the webcomponents.js polyfill loader (for browsers who don't natively support web components), then import the following:
-
-- `d2l-labs-filter-dropdown`
-- `d2l-labs-filter-dropdown-category`
-- `d2l-labs-filter-dropdown-option`
+Import the three filter components:
 
 ```html
-<script src="node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
-<script type="module" src="node_modules/@brightspace-ui-labs/d2l-facet-filter-sort/components/filter-dropdown/filter-dropdown.js"></script>
-<script type="module" src="node_modules/@brightspace-ui-labs/d2l-facet-filter-sort/components/filter-dropdown/filter-dropdown-category.js"></script>
-<script type="module" src="node_modules/@brightspace-ui-labs/d2l-facet-filter-sort/components/filter-dropdown/filter-dropdown-option.js"></script>
+<script type="module">
+	import '@brightspace-ui-labs/facet-filter-sort/components/filter-dropdown/filter-dropdown.js';
+	import '@brightspace-ui-labs/facet-filter-sort/components/filter-dropdown/filter-dropdown-category.js';
+	import '@brightspace-ui-labs/facet-filter-sort/components/filter-dropdown/filter-dropdown-option.js';
+</script>
 ```
```

Comment:
Down under Versioning & Releasing, we typically have [an extra bullet](https://github.com/BrightspaceUI/core#releases):
> Deploy a new package to NPM

Reply:
Right forgot about that!